src/supervisor-api.coffee

```diff
@@ -29,11 +31,22 @@ authenticate = (config) ->
 		.catch (err) ->
 			res.status(503).send("Unexpected error: #{err}")
 
+expressLogger = morgan (tokens, req, res) -> [
+		'SAPI'
+		tokens.method(req, res)
+		url.parse(tokens.url(req, res)).pathname # Strip the apikey from the request
```

Comment:
Should the url already be parsed of part of the `req` object?  If so it'd be good to avoid the work of reparsing it

Reply:
Done.